definable/definable/models/google/gemini.py

```
@@ -0,0 +1,1955 @@
+        if combined_original_content:
+            messages.append(
+                Message(
+                    role="tool",
+                    content=combined_original_content,
+                    tool_name=tool_name,
+                    tool_calls=combined_function_result,
+                    metrics=message_metrics,
+                )
+            )
+
+    def _parse_provider_response(self, response: GenerateContentResponse, **kwargs) -> ModelResponse:
```

Ruff (0.14.14)

[warning] Unused method argument: `kwargs`

(ARG002)